src/common/int-util.h

```diff
@@ -206,6 +209,12 @@ static inline void memcpy_swap64(void *dst, const void *src, size_t n) {
   }
 }
 
+#if _MSC_VER
```

Comment:
This is using #if, the previous one is using #ifndef, are you really trying to test for definedness in one case, and the value of it in the other ?

Reply:
Nice catch, fixed